playbook/app/javascript/kits.js

```diff
@@ -0,0 +1,116 @@
+// !!! IMPORTANT: This file is autogenerated. Please do not edit.!!!
+
```

Comment:
In the old world, these kits were mashed up in the `playbook.js` aka `index.js` file with other imports and things we actually might have wanted to change. Making this it's own auto-generated file prevents human error when then time comes.